test/ui/cypress/CYPRESS.md

```diff
@@ -0,0 +1,72 @@
+# Cypress Testing
+
+Cypress is being used for both end-to-end tests and component tests.
+
+## Prerequisites
+
+1. You have an AAP instance being deployed on Azure.
+2. The Deployment Engine UI is accessible.
+
+## Run
+
+1. Initialize your local environment.
+   You may need to create a virtualenv and install the needed packages from ui/requirements.txt if you don't already have them.
+
+    ```shell
+    cd <repo>/test/ui
+    python -m venv venv
+    source venv/bin/activate    
+
+    # Clean install a project
+    npm ci
+
+    # Verify Cypress installation
+    npx cypress verify
+
+    # Install the following dependencies to see useful messages when the engine is dead.
+    # This is optional, not manadatory.
+    cd <repo>/ui
+    pip install -r requirements.txt
+    ```
+
+2. Configure tests input.
```

Comment:
This section says "tests input" but it is talking about setting up environment variables, not input.

Reply:
@tznamena, in order to get the automated test scenarios running with an existing deployment engine UI, we just need to set those environment variables at OS level or configure them in the cypress configuration file before we kick off Cypress automation to run. That's all.

Reply:
The point is that the environment variables are NOT test input. We need to be clear in our instructions.

Reply:
@tznamena, I updated it to `Configure environment variables for Cypress automation to run`. Hope it is clear now.